src/Utils.hs

```diff
@@ -114,13 +115,13 @@ uniqueShapeName subObjName styShapeName = subObjName ++ nameSep ++ styShapeName
 type Parser = Parsec Void String
 
 rws, attribs, attribVs, shapes, types :: [String] -- list of reserved words
-rws =     ["avoid", "global", "as"] ++ types ++ shapes
+rws =     ["avoid", "global", "as"] ++ types ++ shapes ++ dsll
```

Comment:
`dsll` consists of reserved words for a separate language. Maybe we need to parametrize `rword` function by different sets of reserved words.

Reply:
Agree, I’ll definietly do that. There is not any good reason to mix it up